src/common/daft-config/src/lib.rs

```diff
@@ -26,6 +26,7 @@ pub struct DaftExecutionConfig {
     pub merge_scan_tasks_max_size_bytes: usize,
     pub broadcast_join_size_bytes_threshold: usize,
     pub sample_size_for_sort: usize,
+    pub split_row_groups_max_files: usize,
```

Comment:
We should also expose this to python so it can be set by https://github.com/Eventual-Inc/Daft/blob/5814922581b48eac48ad7941957e1e9187ce1ae4/daft/context.py#L204